signer-service/src/api/services/pendulum.service.js

```diff
@@ -109,14 +127,29 @@ exports.sendStatusWithPk = async () => {
       if (remainingMaxSubsidiesAvailable.lt(SUBSIDY_MINIMUM_RATIO_FUND_UNITS)) {
         isTokensSufficient = false;
         console.log(`Token ${token} balance is insufficient.`);
+
+        slackNotifier.sendMessage({
+          text: `Current balance of funding account is ${nativeToDecimal(
+            tokenBalance,
+          ).toString()} ${token} please charge the account ${fundingAccountKeypair.address}.`,
+        });
       }
     }),
   );
 
   const minimumBalanceFundingAccount = multiplyByPowerOfTen(Big(PENDULUM_FUNDING_AMOUNT_UNITS), apiData.decimals);
   const nativeBalance = Big(balance?.free?.toString() ?? '0');
+
   if (nativeBalance.gte(minimumBalanceFundingAccount) && isTokensSufficient) {
     return { status: true, public: fundingAccountKeypair.address };
   }
+  if (nativeBalance.lt(minimumBalanceFundingAccount)) {
+    slackNotifier.sendMessage({
+      text: `Current balance of funding account is ${divideByPowerOfTen(
```

Comment:
Can we use also `nativeToDecimal` here? or the other way around, but just keep one?